README.md

```diff
@@ -2,6 +2,8 @@
 
 A Django-based website for http://globaltrademotors.com
 
+[![Requirements Status](https://requires.io/github/joshessman/trade-motors/requirements.svg?branch=master)](https://requires.io/github/joshessman/trade-motors/requirements/?branch=master)
+
```

Comment:
Can you please change the url to `[![Requirements Status](https://requires.io/github/sitture/trade-motors/requirements.svg?branch=master)](https://requires.io/github/sitture/trade-motors/requirements/?branch=master)`